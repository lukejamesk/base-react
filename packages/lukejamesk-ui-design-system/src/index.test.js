import { testPackageExports } from '@lukejamesk/test-utils'
import * as lukejameskDesignSystem from '.'

testPackageExports('@lukejamesk/design-system', lukejameskDesignSystem)
