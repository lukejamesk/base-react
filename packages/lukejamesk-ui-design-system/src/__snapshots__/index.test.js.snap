// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`@lukejamesk/design-system should match snapshot 1`] = `
Array [
  "Button",
  "ButtonIcon",
  "ExpansionPanel",
  "ExpansionPanelBody",
  "ExpansionPanelHeader",
  "Icon",
  "Typography",
  "buttonIconVariants",
  "buttonVariants",
  "iconSizes",
  "iconVariants",
  "typographyVariants",
]
`;
